code/game/objects/items/religion.dm

```diff
@@ -1,27 +1,196 @@
 /obj/item/banner
 	name = "banner"
+	desc = "A banner with Nanotrasen's logo on it."
 	icon = 'icons/obj/items_and_weapons.dmi'
 	icon_state = "banner"
 	item_state = "banner"
+	force = 8
+	attack_verb = list("forcefully inspired", "violently encouraged", "relentlessly galvanized")
 	lefthand_file = 'icons/mob/inhands/equipment/banners_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/banners_righthand.dmi'
-	desc = "A banner with Nanotrasen's logo on it."
-	var/moralecooldown = 0
-	var/moralewait = 600
+	var/inspiration_available = TRUE //If this banner can be used to inspire crew
```

Comment:
name/desc/examine/something should indicate to players whether or not this is set